test/integration/verify/controls/azurerm_management_groups.rb

```diff
@@ -0,0 +1,18 @@
+tenant_id = attribute('tenant_id', default: nil)
+
+control 'azurerm_management_groups' do
+  describe azurerm_management_groups do
+    its('group_ids')           { should include "/providers/Microsoft.Management/managementGroups/#{tenant_id}" }
+    its('group_ids')           { should include '/providers/Microsoft.Management/managementGroups/mg_child_one' }
+    its('group_ids')           { should include '/providers/Microsoft.Management/managementGroups/mg_child_two' }
+    its('group_names')         { should include 'mg_parent' }
+    its('group_names')         { should include 'mg_child_one' }
+    its('group_names')         { should include 'mg_child_two' }
+    its('group_types')         { should include '/providers/Microsoft.Management/managementGroups' }
+    its('group_tenant_ids')    { should include tenant_id }
+    its('group_display_names') { should include 'Tenant Root Group' }
```

Comment:
When running this in a fresh environment I see:
```
     ×  Management Groups group_display_names should include "Tenant Root Group"
     expected ["Management Group Child 2", "Management Group Child 1", "Management Group Parent"] to include "Tenant Root Group"
```
So could either add terraform to create the expected 'Tenant Root Group' or simply remove this line maybe?

Reply:
The `Tenant Root Group` is the root group in every Azure account. See picture on [this page](https://docs.microsoft.com/en-us/azure/governance/management-groups/).

If you have 0 created management groups, the root group won't be visible. As soon as you create the first management group, it auto-creates a root group and that newly created group becomes it's child. More details can be found [here](https://docs.microsoft.com/en-us/azure/governance/management-groups/#root-management-group-for-each-directory).

Reply:
I suspect the default permissions don't allow to return this. I'm thinking would be preferable to restrict the test to the management groups that are created in terraform.